Fix #2678 don't add loadmodule when from config

remicollet opened the pull request:
See #2678 

Track in `moduleLoadQueueEntry` where the module was loaded from
* config (moduleEnqueueLoadModule)*
* command (module load or loaded)

Only add `loadmodule` line when not from config to avoid duplicate line.

remicollet committed:
Fix #2678 don't add loadmodule when from config

only protect loadmodule from include files

Signed-off-by: Remi Collet <remi@remirepo.net>

zuiderkwast commented:
I think this is a bugfix and I think we should backport it. I can't see how it's a breaking change in any way.

zuiderkwast commented:
@remicollet Please fix the DCO issue. Essentially, commits need to be done with `git commit -s`. See the DCO failing check details page for information.

remicollet commented:
> @remicollet Please fix the DCO issue. Essentially, commits need to be done with `git commit -s`. See the DCO failing check details page for information.

Smashed and signed

remicollet commented:
> Great, but author and signer emails need to match.

fixed

ljavorsk commented:
Hi, any update on this?

madolson committed:
Update version to 9.1.0-rc1 and add release notes (#3333)

Add release notes for Valkey 9.1 for review and bumping the version so we can stamp it.

---------

Signed-off-by: Madelyn Olson <madelyneolson@gmail.com>
